modules/customizing-the-web-console-URL.adoc

```diff
@@ -3,67 +3,60 @@
 // * web_console/customizing-the-web-console.adoc
 
 [id="customizing-the-web-console-url_{context}"]
-= Customizing the web console URL
+= Customizing console routes
 
-You can update the web console URL, `consoleURL`, to a custom value.
+For `console` and `downloads` routes, custom routes functionally is now implemented in the new `ingress` config API. The Console Operator already contained custom route customization, but for the `console` route only. Now, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration takes precedent, because the route configuration via `console-operator` config is being deprecated.
```

Comment:
```suggestion
For `console` and `downloads` routes, custom routes functionality is now implemented in the new `ingress` config API. The Console Operator already contained custom route customization, but for the `console` route only. Now, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration takes precedent, because the route configuration via `console-operator` config is being deprecated.
```

Reply:
Fixed. Thank you!